src/commands/helm-ct-lint.yaml

```diff
@@ -0,0 +1,18 @@
+---
+description: Lint Helm charts
+parameters:
+  config_file:
+    type: string
+    default: ""
```

Comment:
Let's not specify default in the command.

```suggestion
```

Reply:
My understanding is, that by removing `default` means that the option is required (which it isn't)

Reply:
You're right but we always pass the parameter from the job. I'd rather not bother with keeping defaults in sync in multiple places.